style.css

```diff
@@ -155,25 +164,30 @@ html {
 	justify-content: center;
 	font-family: 'Varela Round', sans-serif;
 	font-size: 3rem;
+	padding-top: 50px;
+	padding-bottom: 75px;
+	margin: 0;
 }
 
 .container {
 	display: flex;
 	align-items: center;
 	justify-content: center;
 	gap: 30px;
-	padding-top: 15px;
-	padding-bottom: 15px;
+	padding-top: 30px;
 	font-family: 'Varela Round', sans-serif;
 	font-weight: bold;
 	color: #2a2e32;
 	font-size: 2rem;
 	height: 40vh;
 }
 
-footer {
-	background-color: #BD4B4B;
-	height: 20vh;
+@media screen and (max-width: 600px) {
```

Comment:
I think you might need to increase the size at which your media query kicks in or reduce the width of those sections, see image: 
![image](https://user-images.githubusercontent.com/45392468/149965163-8a82a340-867e-48d4-8cb2-336f562a47c8.png)

Reply:
runs over the edge of the screen :)